src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -237,6 +237,59 @@ trait GoldTransforms extends SparkSessionWrapper {
       )
   }
 
+  protected def buildVerboseAudit(
```

Comment:
This looks a lot better than all of the logic being in gold -- not sure why the logic is all duplicated -- Please take the requested changes from Gold.scala and implement there here